Cargo.toml

```diff
@@ -11,8 +11,8 @@ categories = ["hardware-support"]
 repository = "https://github.com/aws/aws-nitro-enclaves-nsm-api"
 
 [workspace]
-members = [ ".", "nsm-lib", "nsm-test" ]
-default-members = [ "." ]
+members = [".", "nsm-lib", "nsm-test"]
+default-members = ["."]
```

Comment:
My formatter noticed that the cargo.toml was inconsistent here. Happy to revert these changes if they're view as a problem.